provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -115,14 +120,17 @@ def _get_devices(self, pool: PoolHandle, node: str) -> List[str]:
         return all_cvg_devices
 
     # cluster>storage_set[N]>nodes
-    def _get_server_nodes(self, pool: PoolHandle) -> List[str]:
+    def _get_data_nodes(self, pool: PoolHandle) -> List[str]:
```

Comment:
Please rebase with latest kubernetes branch, there already exist a function called `get_io_nodes()` let's use that in place of `_get_data_nodes()`, result of `_get_data_nodes()` is just used to count the entries. Also we don't need one `_get_data_nodes()` function per file.

Reply:
Since _get_data_nodes is a private function. It would be expecting for pool as an argument. Pool contains value like (cluster_id=cluster_id,pool_type=pool_type,storage_ndx=i)
```
PoolHandle(cluster_id=cluster_id,
                                    pool_type=pool_type,
                                    storage_ndx=i)
```
function call like below:
`hare_config() from main.py ==> generate() from cdf.py ==> _get_cdf_dhall() ==> _create_pool_descriptions() ==> _validate_pool() ==> _get_data_nodes()`

No need to combine this function. Here,  get_io_nodes() will not be very useful since it will be expecting pool as an arguments.

Reply:
@rahul27kumar , as discussed we can devide `get_io_nodes()` into 2 parts, one is to return cluster wide nodes running motr io service and another one that returns nodes in a given storage set running motr io services. This will avoid duplication of code.